app/src/main/java/com/theuhooi/uhooipicbook/modules/monsterlist/MonsterListFragment.kt

```diff
@@ -0,0 +1,59 @@
+package com.theuhooi.uhooipicbook.modules.monsterlist
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.theuhooi.uhooipicbook.R
+import com.theuhooi.uhooipicbook.modules.monsterlist.entity.MonsterContent
+import com.theuhooi.uhooipicbook.modules.monsterlist.entity.MonsterContent.MonsterItem
+
+class MonsterListFragment : Fragment() {
+
+    // MARK: Stored Instance Properties
+
+    private var listener: OnListFragmentInteractionListener? = null
+
+    // MARK: View Life-Cycle Methods
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_monster_list, container, false)
+        if (view is RecyclerView) {
```

Comment:
You can cast `view` to RecyclerView because `view` is always instance of RecyclerView.

```kotlin
val view = inflater.inflate(R.layout.fragment_monster_list, container, false) as RecyclerView
```

Reply:
It was automatically generated, I thought it was good😅
Fixed it.

Reply:
it seems that the cast inserted again, but it's ok if you like the coding style.
https://github.com/uhooi/UhooiPicBook-Android/pull/1/commits/2a22623295cd8949af2ed2c410a4cbb93318e7d7

Reply:
I don't know how to cast.
how's this?
d9801fb

Reply:
A warning occurred when I assigned to a variable of the same name, so I renamed it.
a75cfbc